bin/df/df.c

```diff
@@ -128,11 +128,10 @@ main(int argc, char *argv[])
 			/*
 			 * POSIX specifically discusses the behavior of
 			 * both -k and -P. It states that the blocksize should
-			 * be set to 1024. Thus, if this occurs, simply break
-			 * rather than clobbering the old blocksize.
+			 * be set to 1024.
 			 */
-			if (kflag) {
-			    setenv("BLOCKSIZE", "1024", 1);
+            if (kflag) {
```

Comment:
These changes in indentation are not needed.

Reply:
What I usually do is copy over the original file, open it split-screen and copy it back.